main.go

```diff
@@ -92,13 +94,29 @@ func initConfig() {
 }
 
 func initClient() *http.Client {
+
+	transport := &http.Transport{
+		MaxIdleConnsPerHost:   viper.GetInt("app.http_client.max_idle_conns"),
+		ResponseHeaderTimeout: time.Duration(viper.GetDuration("app.http_client.request_timeout") * time.Millisecond),
+	}
+
+	proxyURLString := viper.GetString("app.http_client.proxy_url")
+
+	if proxyURLString != "" {
+
+		proxyURL, err := url.Parse(proxyURLString)
+		if err != nil {
+			errLog.Fatalf("Error enter proxy_url: %s", err)
```

Comment:
Can you change this to 

```
errLog.Fatalf("Unable to parse `proxy_url`: %s", err)
```

Reply:
I made the change from you. It's true, the message is more meaningful.